src/AppBundle/Controller/DefaultController.php

```diff
@@ -4,15 +4,17 @@
 
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
-use Symfony\Component\HttpFoundation\Request;
```

Comment:
If we keep it, it should at least be in the `indexAction` method to show why its here.